ROBOT_FRAME: "base_link"
MAX_VELOCITY: 0.8
MIN_VELOCITY: 0.0
MAX_YAWRATE: 1.0
MAX_D_YAWRATE: 100.0
MAX_ACCELERATION: 100.0
FRONT_FRAME_DISTANCE: 0.8
REAR_FRAME_DISTANCE: -0.25
LEFT_FRAME_DISTANCE: 0.3
RIGHT_FRAME_DISTANCE: -0.3
